### PR TITLE
[WIP] Add package ncrack

### PR DESCRIPTION
**Pull request by Auxilus**

### CODE REVIEW

**Comment by sdrausty** [comment]
Great work @Auxilus

File: packages/ncrack/build.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+TERMUX_PKG_SHA256=4dd4abfa7c38edafd0da3a3f2ad76f9fae476e147042def87d027879444bd0b9
+TERMUX_PKG_SRCURL=https://github.com/nmap/ncrack/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_BUILD_IN_SRC=yes
+TERMUX_PKG_DEPENDS="openssl, openssh, openssl-dev"
+
+termux_step_post_configure () {
+        echo
+        echo
+        
+        cd /opensshlib/
```
Comment (Neo-Oli) [inline packages/ncrack/build.sh]: The problem is this line not line 18

Comment (Auxilus) [reply packages/ncrack/build.sh]: So what should I do?

Comment (Auxilus) [reply packages/ncrack/build.sh]: The code has to be executed before `make` and after `configure`

**Pull request closed without merging**